### PR TITLE
Add cities to the start page

### PR DESCRIPTION
**Pull request by moonglum**
Added Hamburg to the start page. Also added Ruhr area to the English version :wink:

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -268,7 +268,7 @@ en:
         nothing: Nothing
     welcome:
       deutschland:
-        message: "[Hacken.in](http://hacken.in). Your calendar for events around hacking, geek culture and everything that has to do with being a nerd. At the moments there are events for [Berlin](move_to/berlin), [Cologne](/move_to/koeln) and [Munich](move_to/muenchen)."
+        message: "[Hacken.in](http://hacken.in). Your calendar for events around hacking, geek culture and everything that has to do with being a nerd. At the moments there are events for [Berlin](move_to/berlin), [Cologne](/move_to/koeln) [Munich](move_to/muenchen), [Hamburg](move_to/hamburg) and the [Ruhr area](move_to/ruhrgebiet)."
```
Comment (bascht) [inline config/locales/en.yml]: Inverse Oxford or just a forgotten comma? :)

**Comment by moonglum** [comment]
@bascht Fixed!

**Comment by bascht** [comment]
Beautiful! <3

**Comment by moonglum** [comment]
Thanks!

**Pull request merged**